src/components/views/elements/RoomTopic.tsx

```diff
@@ -71,12 +70,14 @@ export default function RoomTopic({ room, ...props }: IProps): JSX.Element {
                 description: (
                     <div>
                         <Linkify
-                            as="p"
-                            onClick={(ev: MouseEvent) => {
-                                if ((ev.target as HTMLElement).tagName.toUpperCase() === "A") {
-                                    modal.close();
-                                }
+                            options={{
+                                attributes: {
+                                    onClick() {
+                                        modal.close();
```

Comment:
Same question about A tags

Reply:
The old code bound a handler on the parent div, this binds the handler on all the linkified elements (anchor tags)